GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+# Kiali Governance
+
+This document defines governance policies for the Kiali project.
+
+## Maintainers
+
+Kiali Maintainers have write access to the Kiali GitHub repositories https://github.com/kiali.
+They can merge their own patches or patches from others. The current maintainers can be found in https://github.com/orgs/kiali/teams/maintainers.
+
+This privilege is granted with some expectation of responsibility: maintainers are people who care about the Kiali project and want to help it grow and improve. A maintainer is not just someone who can make changes, but someone who has demonstrated his or her ability to collaborate with the team, get the most knowledgeable people to review code, contribute high-quality code, and follow through to fix issues (in code or tests).
+
+A maintainer is a contributor to the Kiali project's success and a citizen helping the project succeed.
+
+## Becoming a Maintainer
+
+To become a maintainer you need to demonstrate the following:
+
+  * commitment to the project
+    * participate in discussions, contributions, code reviews for 3 months or more,
+    * perform code reviews for 10 non-trivial pull requests,
+    * contribute 10 non-trivial pull requests and have them merged into master,
+  * ability to write good code,
```

Comment:
I wonder if "good" is the right term here, "good" is subjective. Perhaps something about well structured, clear, maintainable, documented and tested?

Reply:
To me "good" is as good and subjective as "structured, clear, maintainable, documented and tested".
Both expressions are similar and subjective.

Reply:
+1 I'd change to, "the ability to write high quality code."